src/components/backend-ai-session-launcher.ts

```diff
@@ -3359,6 +3359,29 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
             </wl-expansion>
           </div>
           <div id="progress-04" class="progress center layout fade">
```

Comment:
Let's add the type of session (`BATCH` or `INTERACTIVE`).
And I cannot see proper session info when I enabled two keys `debug` or `allowManualImageNameForSession` to true in configuration file(`config.toml`). Please update when the user uses the manual image.

**Screenshot**:
<img width="432" alt="using-manual-image-in-session-launcher-01" src="https://user-images.githubusercontent.com/46954439/149491911-fac9ec87-631a-41a8-a68f-d6cbd1bde4af.png">
<img width="432" alt="using-manual-image-in-session-launcher-02" src="https://user-images.githubusercontent.com/46954439/149491924-fcea7841-bf99-4fc9-a51b-67c3058b89be.png">

As you can see the pictures above, there's no image description and the environment doesn't match with logo.